test/index.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var assert = require('assert');
+
+var Z = require('sanctuary-type-classes');
```

Comment:
This should be fine for testing/scripting since `type classes` will depend on `type identifiers` as a dependency (I believe?) at some point and `type classes` would load it's dependency from /node_modules/ instead of the local version.

So you shouldn't have a case were a breaking change in development would cause Z to be undefined.